### PR TITLE
Add ramda.traverse

### PR DESCRIPTION
**Pull request by hojberg**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ x Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://ramdajs.com/docs/#traverse
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/ramda/index.d.ts [hunk 1]
```diff
@@ -1712,6 +1713,15 @@ declare namespace R {
          */
         transpose<T>(list: T[][]): T[][];
 
+		/**
+		 * Maps an Applicative-returning function over a Traversable, then uses
+		 * sequence to transform the resulting Traversable of Applicative into
+		 * an Applicative of Traversable.
+		 */
+		traverse<T, U, A>(of: (a: U[]) => A, fn: (t: T) => U, list: T[]): A;
```
Comment (andy-ms) [inline types/ramda/index.d.ts]: Mixes tabs and spaces

**Comment by hojberg** [comment]
@andy-ms that should be good now :)

**Comment by andy-ms** [comment]
Awaiting feedback from reviewers.

**Comment by 1M0reBug** [comment]
Sounds good to me, not the easiest function.

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**